### PR TITLE
DOCSP-25363 Compass command line options

### PR DESCRIPTION
**Pull request by jocelyn-mendez1**
Compass now accepts command line options.

Currently I have placed this info here but the goal is to add this as sub-page of this PR: https://github.com/mongodb/docs-compass/pull/435

JIRA: https://jira.mongodb.org/browse/DOCSP-25363

STAGING:https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25363/command-line-options/

### CODE REVIEW

File: source/connect/cli-options.txt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.. _cli-options:
+
+----------------------------
+Compass Command Line Options
+----------------------------
+
+The following page provides an overview on how set command line options for
```
Comment (jmd-mongo) [inline source/connect/cli-options.txt:7]: [missing word]

I think we're missing "to" on line 7.

Comment (jocelyn-mendez1) [reply source/connect/cli-options.txt]: Thank you and done!

File: source/connect/cli-options.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+modified on the |compass-short| interface.
+
+Syntax
+------
+
+Use the following syntax to start |compass| and set a configuration option:
+
+.. code-block:: sh 
+
+    <path-to-Compass-executable> --<option>
```
[the remark this answers is not in the document]
Comment (jocelyn-mendez1) [reply source/connect/cli-options.txt:26]: I actually realized this is not unique to Mac and should work for Windows and Linux!

**Review by jmd-mongo — APPROVED** [review]
This one LGTM. Thanks for the updates, @jocelyn-mendez1!

File: source/connect/cli-options.txt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Options
+-------
+
+You can set the following configuration options using the command line:
```
Comment (addaleax) [inline source/connect/cli-options.txt:31]: Note that https://github.com/mongodb/docs-compass/pull/442 starts to document other command line options already, so whichever pull request is merged second probably needs to be updated to expand this list here cc @davemungo

**Review by davemungo — COMMENTED** [review]
@jocelyn-mendez1  - did you want to add the command line example as well? 

https://www.mongodb.com/docs/compass/master/connect/connect-from-the-command-line/#command-line-connection-specification

File: source/includes/compass-executable.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Windows
+~~~~~~~
+
+The executable is called ``MongoDBCompass.exe``. The installer installs
+it in a folder you pick during the installation process.
+
+MacOS
+~~~~~
+The executable is called ``Compass``. The installer installs
```
Comment (jmd-mongo) [inline source/includes/compass-executable.rst:17]: I don't think this is quite correct. I think the executable is technically `./"MongoDB Compass"`.

For example, calling `Compass` does not work on my machine:

```
[joe! /Applications/MongoDB Compass/Contents/MacOS]$ ./"Compass"
-bash: ./Compass: No such file or directory
```

However, this opens Compass:

```
[joe! /Applications/MongoDB Compass/Contents/MacOS]$ ./"MongoDB Compass"
```

**Review by jmd-mongo — COMMENTED** [review]
Overall this is looking good -- I left a note about syntax re calling Compass from the command line on macOS!

Thanks,
Joe

File: source/includes/compass-executable.rst (hunk 4 above)
Comment (jocelyn-mendez1) [reply source/includes/compass-executable.rst:17]: Done.

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update! LGTM!

File: source/includes/compass-executable.rst (hunk 4 above)
Comment (jmd-mongo) [reply source/includes/compass-executable.rst]: Thanks!

**Pull request merged**